wp-content/themes/goonj-crm/assets/styles/lg.scss

```
@@ -58,6 +58,21 @@
         .error {
             font-size: 14px;
         }
+
+        .af-container:not(:has(.af-container)) {
+            display: flex !important;
+            flex-wrap: wrap !important;
+        }
+    
+        .af-container af-field {
```

I think `af-field` is a class and hence is missing a dot `.`
```suggestion
        .af-container .af-field {
```

---

@pokhiii The **af-field** is a custom tag, not a class in this case.

---

Ah, my bad.